docs/messaging/kafka.md

```diff
@@ -8,7 +8,12 @@ linkTitle: Kafka
 
 <!-- toc -->
 
-- [Span attributes](#span-attributes)
+- [Attributes](#attributes)
+- [Metrics](#metrics)
+  * [Metric: `messaging.publish.duration`](#metric-messagingpublishduration)
```

Comment:
I wonder, didn't we had a person working on the Kakfa metrics and decided to not have them in semconv? How these will fit with those? Or are those broker and these are client ones?